lib/workload/stateless/stacks/fastq-manager/Readme.md

```diff
@@ -0,0 +1,416 @@
+# Fastq Manager
+
+Welcome to the fastq manager, where we, well manage fastqs!!
+
+The fastq manager sits over the file manager and provides a way to easily access fastq files and their metadata. 
+
+We provide a few use-cases to consider
+
+1. Given a library to perform an analysis, return the appropriate fastq files for that library.\
+  * This might be in CWL input format, or a list of file paths, or a list of presigned urls.
+
+2. Given a fastq id, return the metadata associated with that file.
+  * This might be 
+    * library, 
+    * sample name, 
+    * instrument run, 
+    * lane, 
+    * size, 
+    * read count
+    * the md5sum, 
+    * the url, etc.
+
+3. Search for fastq files given a project, subject, instrument run id or other metadata.
+
+We also provide endpoints to run qc, ntsm and file metric calculations on fastq files (like raw md5sum and gzip file size in bytes), which are then 
+stored in the database.
+
+We also provide an endpoint to compare two fastq files with ntsm and compare the results. 
+
+
+## Events generated
+
+We generate events after any 'PATCH' or 'POST' request to the fastq manager.
+
+These events are sent to the event bus and can be consumed by other services.
+
+Examples of events generated are:
+
+### Fastq Created Events
+
+```json5
+{
+  "EventBusName": "OrcaBusMain",  // Name of our event bus
+  "Source": "fastq.manager",  // lowercase name of the service
+  "DetailType": "FastqListRowCreated",  // PascalCase name of the event
```

Comment:
Much better event name, good job!

Just minor note on source field, the convention is `orcabus.fastqmanager`